Ukol_1/python/vypocty.py

```diff
@@ -43,13 +47,22 @@ def median(cisla: list[int]) -> float:
     Returns:
       Medián čísel.
     """
-    pass  # TODO: Implementujte funkci
+    cisla.sort()
+    delka = len(cisla)
+    if delka % 2 == 0:
+        return (cisla[delka // 2 - 1] + cisla[delka // 2]) / 2
```

Comment:
Tady dostávám list out of range error pokud přidám prázdný list poprosím o kontrolu prázdného listu a následně zavolání `raise ValueError`